src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -292,6 +288,25 @@ fluid.textToSpeech.handleEnd = function (that) {
     }
 };
 
+// Find the voice by the language code
+// 1. If the language code is not provided, use the default language;
+// 2. If the voice for the language code is not found, fall back to the first voice that supports the same country
+// code;
+// 3. If the voice is still not found, return the voice of options.defaultLanguage.
+fluid.textToSpeech.getVoiceByLang = function (that, lang) {
```

Comment:
Dependence on the compoinent is minimal and might work better with "voices" and "defaultLanguage" hoisted into arguments